### PR TITLE
Add hook-gcloud.py

### PR DESCRIPTION
**Pull request by happyeric77**
Add hook-cloud.py: pyinstaller not working with pyrebase google module.

Fixes [#5316](https://github.com/pyinstaller/pyinstaller/issues/5316).

### CODE REVIEW

File: news/68.new.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add a hook for ``gcloud``
```
Comment (bwoodsend) [inline news/68.new.rst:1]:
```suggestion
Add a hook for ``gcloud`` which requires its distribution metadata.
```

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gcloud.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import copy_metadata
+datas = copy_metadata('gcloud')
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gcloud.py:14]: Can we have a trailing newline to keep the linter happy.

**Review by bwoodsend — CHANGES_REQUESTED** [review]
We need a news entry. See the README

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gcloud.py (hunk 2 above)
Comment (happyeric77) [reply src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gcloud.py]: Tks for the advice. let me make a change.

**Comment by bwoodsend** [comment]
Perfect.

**Pull request merged**